<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>默陌老屋手迹</family>
     <pointsize>48</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>五子棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_player">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>人人对战</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_aid">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>人机对战(高阶)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_aie">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>人机对战(低阶)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
